tools/lib/helpers.py

```diff
@@ -5,12 +5,16 @@
 
 # regex patterns
 class RE:
-  DONGLE_ID =  r'(?P<dongle_id>[a-z0-9]{16})'
-  TIMESTAMP = r'(?P<timestamp>[0-9]{4}-[0-9]{2}-[0-9]{2}--[0-9]{2}-[0-9]{2}-[0-9]{2})'
-  LOG_ID_V2 = r'(?P<count>[a-z0-9]{8})--(?P<uid>[a-z0-9]{10})'
-  LOG_ID = r'(?P<log_id>(?:{}|{}))'.format(TIMESTAMP, LOG_ID_V2)
+  DONGLE_ID_BASE = '[a-z0-9]{16}'
+  DONGLE_ID =  r'(?P<dongle_id>{})'.format(DONGLE_ID_BASE)
+  LOG_ID_BASE = r'[a-z0-9-]{20}'
```

Comment:
why drop the old patterns? now it's missing the validation and the groups, which are very useful

Reply:
Reduces complexity and we currently have no examples of where what the format is matters once we are not relying on it as a timestamp.